drafts/builtins-partiql.adoc

```diff
@@ -0,0 +1,250 @@
+= PartiQL Builtins Support
```

Comment:
Does it makes sense to link to the `builtins-sql` document here clarifying that the first column is derived from there?

Reply:
Yes, I will add this